### ISSUE TITLE
add table finished but the table state is unknown

### ISSUE THREAD

**Issue by 3AceShowHand** [pingcap/tiflow#9250]
```
[2023/06/14 03:24:36.869 +00:00] [INFO] [processor.go:261] ["Add Table finished"] [captureID=9c013e02-de86-436d-809f-219a266cd66a] [namespace=default] [changefeed=kafka-random-broker-fail-canal-json] [span={table_id:105,start_key:7480000000000000ff695f720000000000fa,end_key:7480000000000000ff695f730000000000fa}] [tableResolvedTs=442161712796795385] [tableCheckpointTs=442161689072500772] [globalCheckpointTs=442161689072500772] [state=Unknown] [isPrepare=false]

```

**Comment by nongfushanquan** [comment]
Could you please describe the result of this issue? Will it cause ticdc to panic?

**Comment by 3AceShowHand** [comment]
> Could you please describe the result of this issue? Will it cause ticdc to panic?

It won't cause the CDC server to fail, just a confusing log.

**Comment by 3AceShowHand** [comment]
closed by #9276
